### PR TITLE
Add validator for kubelet anonymous auth disabled

### PR DESCRIPTION
**Pull request by Cynerva**
This tests https://github.com/juju-solutions/bundle-canonical-kubernetes/issues/314 by verifying that kubelet returns a 401 Unauthorized response to anonymous clients.

### CODE REVIEW

**Comment by Cynerva** [comment]
Don't merge yet, this depends on https://github.com/juju/python-libjuju/pull/153

File: tests/upgrade-tests/validation.py [hunk 1]
```diff
@@ -71,3 +73,14 @@ def validate_status_messages(model):
     assert action.status == 'completed'
     # TODO: wait for pods running
     # TODO: test that we can reach the ingress endpoint
+
+
+async def validate_kubelet_anonymous_auth_disabled(model):
+    ''' Validate that kubelet has anonymous auth disabled '''
+    units = model.applications['kubernetes-worker'].units
+    for unit in units:
```
Comment (Cynerva) [inline tests/upgrade-tests/validation.py]: Pushed an update to do this, thanks

**Comment by tvansteenburgh** [comment]
LGTM. +1 once libjuju bits are available.

**Comment by Cynerva** [comment]
Libjuju bits are released. I'mma go ahead and merge since this has a LGTM

**Pull request merged**